examples/mesos/minimesos.json

```diff
@@ -0,0 +1,51 @@
+{
+  "id": "weave-scope",
+  "cpus": 1,
+  "mem": 128,
+  "instances": 1,
+  "constraints": [
+    [
+      "hostname",
+      "UNIQUE"
+    ]
+  ],
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "image": "weaveworks/scope:0.15.0",
+      "network": "HOST",
+      "privileged": true,
+      "parameters": [
+        {
+          "key": "pid",
+          "value": "host"
+        },
+        {
+          "key": "name",
+          "value": "weavescope"
+        }
+      ]
+    },
+    "volumes": [
+      {
+        "containerPath": "/var/run/docker.sock",
+        "hostPath": "/var/run/docker.sock",
+        "mode": "RW"
+      }
+    ]
+  },
+  "args": [
+    "--probe.docker",
+    "true"
+  ],
+  "env": {
+    "CHECKPOINT_DISABLE": ""
```

Comment:
@errordeveloper you had a valid point. After reading https://www.weave.works/documentation/scope-latest-how-it-works/ I set `"CHECKPOINT_DISABLE": "true"`